packages/panzer/mini-em/example/BlockPrec/CMakeLists.txt

```diff
@@ -79,6 +79,18 @@ TRIBITS_ADD_TEST(
    NUM_MPI_PROCS 4
    )
 
+IF(NOT Kokkos_ENABLE_Cuda AND NOT Kokkos_ENABLE_OpenMP)
+
+  TRIBITS_ADD_TEST(
+    BlockPrec
+    NAME "MiniEM-BlockPrec_RefMaxwell2D"
+    ARGS "--solver=MueLu-RefMaxwell --numTimeSteps=1 --linAlgebra=Tpetra --spatialDim=2"
```

Comment:
What's preventing 2D from being run with cuda and openmp?

Reply:
Nothing but my laziness. Currently, MiniEM has one input XML per configuration. Things change depending on spatial dimension, underlying linear algebra library, node type and solver. I'm starting to think that it might be easier to only save the differences compared to some default XML, and apply the one after the other. E.g. apply one for 2D problems, and another one for node type OpenMP.